Modix/ModixBot.cs

```diff
@@ -168,6 +166,10 @@ public ModixBot(ModixConfig config, ILogger logger)
             _map.AddSingleton<IPromotionRepository, DBPromotionRepository>();
 
             _map.AddSingleton<CommandErrorHandler>();
+            _map.AddSingleton<InviteLinkHandler>();
```

Comment:
You should wrap these up in an extension method `AddBehaviourConfiguration` on a class `BehaviourConfigurationSetup` in `Modix.Services.BehaviourConfiguration`. That's what I've done consistently in #91.